script.js

```diff
@@ -1,121 +1,145 @@
-var input = document.querySelectorAll('input');
+var myLibrary = [];
```

Comment:
Variables that don't change in the code lifecycle should always be declared with `const`. That enforces immutability which helps your code be more stable.